cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -150,7 +162,13 @@ def index_item(item, skip_index=False):
                 skip_child_index = skip_index or \
                     (triggered_at is not None and (triggered_at - item.subtree_edited_on) > reindex_age)
                 for child_item in item.get_children():
-                    index_item(child_item, skip_index=skip_child_index)
+                    item_content_groups = getattr(item, 'content_groups', None)
+                    if item_content_groups:
+                        index_item(child_item, skip_index=skip_child_index,
```

Comment:
Nitpick / Optional: once the line is long enough, I prefer to place each argument on it's own line like this:
```
    index_item(
        child_item,
        skip_index=skip_child_index,
        groups_usage_info=groups_usage_info,
        content_groups=item.content_groups
    )
```